### ISSUE TITLE
(P4 texts) Elegy and Iambus

### ISSUE THREAD

**Issue by lcerrato** [PerseusDL/canonical-greekLit#1220]
Is there a migration plan for these files? It appears they were not moved because they are composite docs.

http://www.perseus.tufts.edu/hopper/text?doc=Perseus%3atext%3a2008.01.0477
http://www.perseus.tufts.edu/hopper/text?doc=Perseus%3atext%3a2008.01.0478
http://www.perseus.tufts.edu/hopper/text?doc=Perseus%3atext%3a2008.01.0479
http://www.perseus.tufts.edu/hopper/text?doc=Perseus%3atext%3a2008.01.0480

**Comment by lcerrato** [comment]
@AlisonBabeu 
These were assigned a VIAF instead of conventional URNs and I do not see the files.
https://docs.google.com/spreadsheets/d/1ioyT7_RItEN3y309ot3-ls10FoPI3x3aWJLRavyxYUk/edit?usp=sharing

Was it because of the composite nature? Not sure what to do.

**Comment by AlisonBabeu** [comment]
hi @lcerrato I'm pretty sure that was why because there was no plan at the time to break the XML files into individual MODS records for the author/works although I do have individual metadata on all these authors/works in other catalog records.  I believe these 3 volumes are also not found anywhere off of P4 I believe because of uncertain copyright status?

**Comment by lcerrato** [comment]
Yes, looks like a print date rather than copyright, but a good point regardless. I can move ahead with a viaf or another assigned ID if desired.
